next.config.js

```diff
@@ -4,6 +4,9 @@ const nextConfig = {
   compiler: {
     emotion: true,
   },
+  images: {
+    domains: ['raw.githubusercontent.com'],
+  },
```

Comment:
Images coming from the API could be from dynamic and unknown sources, not necessarily from `githubusercontent.com`, so the in order to prevent crashes would be better to handle the images as `unoptimized`, this way we can still use the `Image` component and render properly any image of any source safely, this next config is not necessary using that approach

Reply:
Got it, check now